robyn/__init__.py

```diff
@@ -165,6 +166,40 @@ def terminating_signal_handler(_sig, _frame):
                 observer.stop()
                 observer.join()
 
+    def add_view(self, endpoint: str, view: Callable, const: bool = False):
+        """
+        [This is base handler for the view decorators]
+
+        :param endpoint [str]: [endpoint for the route added]
+        :param handler [function]: [represents the function passed as a parent handler for single route with different route types]
+        """
+        http_methods = {"GET", "POST", "PUT", "DELETE", "PATCH", "HEAD", "OPTIONS"}
+
+        def get_functions(view):
+            functions = get_all_nested(view)
+            output = []
+            for name, handler in functions:
+                route_type = name.upper()
+                if route_type in http_methods:
+                    output.append((route_type.upper(), handler))
```

Comment:
`route_type` is already upper case
```suggestion
                    output.append((route_type, handler))
```

Reply:
Right, perfect.